hack/make-rules/test-cmd-util.sh

```diff
@@ -3924,7 +3924,7 @@ run_cmd_with_img_tests() {
 
   # Test that a valid image reference value is provided as the value of --image in `kubectl run <name> --image`
   output_message=$(kubectl run test1 --image=validname)
-  kube::test::if_has_string "${output_message}" 'deployment.apps "test1" created'
+  kube::test::if_has_string "${output_message}" 'deployment.extensions/test1 created'
```

Comment:
This is not acceptable, we cannot break backwards compatibility. Move the run changes to a separate PR and I'll have a closer look once again on Monday.